tests/proot-test.nix

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2019-2021, see AUTHORS. Licensed under MIT License, see LICENSE.
+
+{ pkgs ? import <nixpkgs> { } }:
+
+let
+  #stdenv = pkgs.pkgsStatic.stdenvAdapters.makeStatic pkgs.stdenv;
+  stdenv = pkgs.stdenv;
+in
+
+stdenv.mkDerivation {
```

Comment:
This proot should be the same version + nixpkgs rev like the on-device one except for the static and cross compiling stuff. Maybe for now you can just somehow import the definition from `pkgs/` to keep it consistent.

Reply:
Indeed. Extracted common stuff into `pkgs/proot-termux` used from both places, hope I've also passed the same nixpkgs as well.